.gitignore

```diff
@@ -1,2 +1,19 @@
+logs
+*.log
+npm-debug.log*
+yarn-debug.log*
+yarn-error.log*
+lerna-debug.log*
+.DS_Store
+
+*.iml
+.idea
+.tscache
+node_modules/
+typings/
+coverage/
+package-lock.json
```

Comment:
@PCalape Remove `package-lock.json` from `.gitignore`, we should always commit this file.

Reply:
Copy on this @arnellebalane. Will apply next commit